README.md

```diff
@@ -343,6 +343,9 @@ Using a non `true monospace font` can lead to visual problems on a `BrailleCanva
 Either change the font to e.g. [JuliaMono](https://juliamono.netlify.app/) or use `border=:dotted` keyword argument in the plots.
 For a `Jupyter` notebook with the `IJulia` kernel see [here](https://juliamono.netlify.app/faq/#can_i_use_this_font_in_a_jupyter_notebook_and_how_do_i_do_it).
 
+Terminals seem to respect a standard aspect ratio of `4:3`, hence a square matrix will often no look square in the terminal.
+As a fix, you can pass `correct_aspect_ratio=true` to `spy` or `heatmap` in order to recover a unit aspect ratio.
```

Comment:
This seems to be an experimental flag so might be helpful to mark it as experimental.

```suggestion
(Experimental) Terminals seem to respect a standard aspect ratio of `4:3`, hence a square matrix does not often look square in the terminal.
As a fix, you can pass `correct_aspect_ratio=true` to `spy` or `heatmap` in order to recover a unit aspect ratio.
This keyword is experimental and might be unnecessary in future versions.
```

Reply:
I dislike the word `correct_aspect_ratio` it's too long. Maybe you have an idea for something shorter ? `fix_ar` ? `fix_43` ?